http-api-data.cabal

```diff
@@ -28,8 +28,8 @@ tested-with:
 
 custom-setup
   setup-depends:
-    base >= 4.7 && <4.11,
-    Cabal >= 1.18 && <2.1,
+    base >= 4.7 && <4.12,
+    Cabal >= 1.18 && <2.2,
```

Comment:
And as I pointed in other issue (iirc `cabal-doctest`) the next proper release of `Cabal` will be `2.2` not `2.1`.

Reply:
Thanks, I'll fix this.